content/en/docs/reference/config/analysis/ist0109/index.md

```diff
@@ -3,10 +3,90 @@ title: ConflictingMeshGatewayVirtualServiceHosts
 layout: analysis-message
 ---
 
-This message occurs when Istio detects an overlap between virtual service
-resources. For example, you have multiple virtual services defined to use the
-same hostname and gateway, where the gateway is set to `mesh` or no gateway
-specified.
+This message occurs when Istio detects an overlap between
+[virtual service](/docs/reference/config/networking/virtual-service)
+resources that conflict with one another. For example, multiple virtual
+services defined to use the same hostname and attached to mesh gateway
+will generate this message. Note that Istio supports merging of virtual
+services that are attached to the ingress gateways.
 
-To resolve this problem, check your Istio configuration, remove the conflicting
-values and try again.
+## Resolution
+
+To resolve this issue, you can take one of the following actions:
+
+* Merge the conflicting virtual services into a single resource.
+* Make the hostnames unique across virtual services attached to mesh gateway.
+* Scope the resource to a specific namespace by setting the `exportTo` field.
+
+## Examples
+
+The following `VirtualServices`, `productpage` in namespaces `team1` and `team2`
+are in conflict as they are attached to the default mesh gateway, and have the
+same host `productpage.default.svc.cluster.local` defined.
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: productpage
+  namespace: team-1
+spec:
+  hosts:
+  - productpage.default.svc.cluster.local
+  http:
+  - route:
+    - destination:
+        host: productpage
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: productpage
+  namespace: team-2
+spec:
+  hosts:
+  - productpage.default.svc.cluster.local
+  http:
+  - route:
+    - destination:
+        host: productpage.team-2.svc.cluster.local
+---
+{{< /text >}}
+
+You can resolve this issue by scoping these `VirtualServices` to their respective namespaces like this:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: productpage
+  namespace: team-1
+spec:
+  exportTo:
+  - "."
+  hosts:
+  - productpage.default.svc.cluster.local
+  http:
+  - route:
+    - destination:
+        host: productpage
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: productpage
+  namespace: team-2
+spec:
+  exportTo:
+  - "."
+  hosts:
+  - productpage.default.svc.cluster.local
+  http:
+  - route:
+    - destination:
+        host: productpage.team-2.svc.cluster.local
+---
+{{< /text >}}
+
+Note that setting the `exportTo` field to `.` restricts the scope to the
+namespace where the virtual service resource exists.
```

Comment:
I'd move this up to where you explain the resolution. e.g. 

```
You can resolve this issue by setting the `exportTo` field to `.` so that each virtual service 
is scoped only to its own namespace. 
```